packages/@icon-magic/distribute/src/create-icon-template.ts

```diff
@@ -20,8 +20,12 @@ export async function createHbs(
     const contents = await asset.getContents();
     // Parse XML from a string into a DOM Document.
     const xml = doc.parseFromString(contents as string, 'image/svg+xml');
-    const id = xml.documentElement.getAttributeNode('id');
+    const el = xml.documentElement;
+    const id = el.getAttributeNode('id');
     const iconName = id ? id.value : '';
+    el.removeAttribute('id');
```

Comment:
Can we remove this from the svg itself during build? I'm guessing this existed for sprite generation and addition of an id can be moved there.

Reply:
where do we store the value of the ID to add? Seems redundant to strip it, save it as another variable then add it again

Reply:
I was thinking we should construct it from the name of the file. That way we aren't touching the internals of the svg during distribution. And we won't be stripping it as such, we'll have to disable the config in svgo, no?

Reply:
If we removed the IDs during generate, we would still need to add the ID (and manipulate the svg) when we distribute to the sprite. We need the ID for the sprite and we don't need the ID for the template.